catroid/src/org/catrobat/catroid/pocketmusic/ui/TrackView.java

```diff
@@ -0,0 +1,58 @@
+/*
+ * Catroid: An on-device visual programming system for Android devices
+ * Copyright (C) 2010-2015 The Catrobat Team
+ * (<http://developer.catrobat.org/credits>)
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU Affero General Public License as
+ * published by the Free Software Foundation, either version 3 of the
+ * License, or (at your option) any later version.
+ *
+ * An additional term exception under section 7 of the GNU Affero
+ * General Public License, version 3, is available at
+ * http://developer.catrobat.org/license_additional_term
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+ * GNU Affero General Public License for more details.
+ *
+ * You should have received a copy of the GNU Affero General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+package org.catrobat.catroid.pocketmusic.ui;
+
+import android.content.Context;
+import android.util.AttributeSet;
+import android.widget.TableLayout;
+
+import java.util.ArrayList;
+import java.util.List;
+
+public class TrackView extends TableLayout {
+
+	public static final int ROW_COUNT = 13;
+	private List<TrackRowView> trackRowViews = new ArrayList<>(ROW_COUNT);
+
+	public TrackView(Context context) {
```

Comment:
The second constructor is needed by Android to inflate the TableLayout, as it is currently used in catroid/res/layout/activity_pocketmusic.xml. The first constructor is intended for programmatically instantiating a TableLayout, which is not used right now (but it will very likely be used in future) so it could for now be deleted.